flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -1655,28 +1655,58 @@ void testSlotSharingResourceConfigurationWithDefaultSlotSharingGroup() {
     @Test
     void testNamingOfChainedMultipleInputs() {
         String[] sources = new String[] {"source-1", "source-2", "source-3"};
-        JobGraph graph = createGraphWithMultipleInputs(true, sources);
-        JobVertex head = graph.getVerticesSortedTopologicallyFromSources().iterator().next();
-        assertThat(sources).allMatch(source -> head.getOperatorPrettyName().contains(source));
+        String sink = "sink";
+        JobGraph graph = createGraphWithMultipleInputs(true, sink, sources);
+        Iterator<JobVertex> iterator = graph.getVerticesSortedTopologicallyFromSources().iterator();
+
+        JobVertex multipleVertex = iterator.next();
+        verifyJobVertexName(multipleVertex, sources, new String[] {sink});
```

Comment:
I think we can directly check the job vertex name by:
 `assertThat(multipleVertex.getName()).isEqualTo("mit [Source: source-1, Source: source-2, Source: source-3]")`.
The same for others.

Reply:
done